### PR TITLE
update readme setup instructions

### PR DESCRIPTION
**Pull request by Omurshid**
Update to use foundry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,22 +41,15 @@ yarn install
...
-
-```sh
-solc-select install 0.8.16
-solc-select use 0.8.16
-```
-
-### Husky
+### Foundry
 
-Install pre-commit hooks: `yarn husky install`
```
Comment (larryob) [inline README.md:59]: I think this is still needed, since we want the pre-commit hooks to run.

File: README.md [hunk 2]
```diff
@@ -72,6 +61,12 @@ $ yarn build
 
 ### Test
 
+Run the foundry local node
```
Comment (larryob) [inline README.md:64]: This should say something like "pin foundry version".

File: README.md [hunk 3]
```diff
@@ -92,22 +87,6 @@ Run solhint:
...
-
-To run a script run `yarn script <script>`. Pass in the ethereum and polygon networks you want to use with the `-eth` and `-p` flags. The two network names will be in the `ETH_NETWORK` and `POLYGON_NETWORK` environment variables. In the script, use `hre.changeNetwork` to change the current network. If run without the `--no-fork` flag `yarn script` will bring up two anvil nodes, one which forks the selected ethereum network, and one which forks the selected polygon network. If run with `--relay` then OZ Relayer will be used to send any transactions.
-
-To run the rebalance script against testnets:
-
-```sh
-$ yarn script scripts/rebalance.ts -eth goerli -p mumbai --no-fork
-```
-
 ### Deploying the Contracts
```
Comment (larryob) [inline README.md:90]: You can get rid of this section as well.

**Review by larryob — COMMENTED** [review]
Looks good, could use a couple of tweaks.

**Comment by tmoon** [comment]
abandoned

**Pull request closed without merging**